python/paddle/distributed/passes/auto_parallel_replace_with_parallel_cross_entropy.py

```diff
@@ -57,9 +60,13 @@ def _check_user(self, value):
     def _apply_single_impl(self, main_program, startup_program, context):
         del_ops = []
         new_ops = []
-        ring_id = self.model_parallel_group.id
-        rank = self.model_parallel_group.rank
-        nranks = self.model_parallel_group.nranks
+
+        params = main_program.global_block().all_parameters()
+        process_ids = params[0].dist_attr().process_mesh.process_ids
```

Comment:
Why use `process_ids` of params[0] instead of the input of 'pd_op.cross_entropy_with_softmax'?

Reply:
get it, I will modify this immediately